src/Microsoft.AspNetCore.HttpOverrides/project.json

```diff
@@ -24,10 +24,11 @@
   "dependencies": {
     "Microsoft.AspNetCore.Http.Extensions": "1.1.0-*",
     "Microsoft.Extensions.Logging.Abstractions": "1.1.0-*",
-    "Microsoft.Extensions.Options": "1.1.0-*"
+    "Microsoft.Extensions.Options": "1.1.0-*",
+    "System.Collections.Immutable": "1.2.0"
```

Comment:
Decided it on my own based on the fact that we don't expose immutable* anywhere else in all of asp.net. Also decided it based on the fact that interfaces should be preferred over concrete types like this. It's the same reason we don't expose List in public API. There's probably a guideline somewhere though, but you can take my word for it 😆